### ISSUE TITLE
Add the option to load the type model definition from file

### ISSUE THREAD

**Issue by b1zzu** [aerogear/graphql-testx#97]
It would be nice if it could be possible to do something like this:
```ts
new TestxServer(path.resolve(__dirname, 'schema.graphql'))
```
but because both the model definition and the file path are strings, in my opinion, something like this is more clear:
```ts
new TestxServer({file: path.resolve(__dirname, 'schema.graphql')})
```
or we could expose a helper method:
```ts
testxServerFromFile(path.resolve(__dirname, 'schema.graphql'))
```

**Comment by vinifarias** [comment]
@b1zzu I really like this feature, we can try to do something like constructor overload.

**Comment by b1zzu** [comment]
Blocked by: https://github.com/aerogear/graphql-testx/issues/103

**Comment by b1zzu** [comment]
@vinifarias I've opened a separate issue to discuss how we could improve the TestxServer constructor/api, we can move the discussion there. Btw what's a constructor overload?

**Comment by vinifarias** [comment]
@b1zzu It's when you use multiple constructors for a class, and each of them has different parameters.

**Comment by b1zzu** [comment]
@vinifarias that's not possible in typescript/javascript, the alternative in our case would be to add n optional parameters, but it has this drawback 
```ts
new TestxServer("option one", undefined, undefined, undefined, undefined, "som other options")
```

**Comment by vinifarias** [comment]
@b1zzu Sorry XD, it's not a good idea to keep this type of constructor with many optional parameters.

**Comment by b1zzu** [comment]
@vinifarias no need to be sorry :smile:, in Java it's a good approach because it's possible to define multiple constructors with the same number of arguments

**Comment by wtrocki** [comment]
I plan to use testx with graphback that will use graphql-config. 

GraphQL Config always passing compiled schema/string. Files are hard as all depends on the __dirname and process.cwd
